src/segwit_addr.c

```diff
@@ -54,7 +54,7 @@ int bech32_encode(char *const output, const size_t out_len, const char *const hr
             chk = bech32_polymod_step(chk) ^ (hrp[i] >> 5);
             ++i;
         }
-        if (i + 7 + data_len > 108)
+        if (i + 7 + data_len > 1023)
```

Comment:
Could be interesting to have a comment explaining this hardcoded value...

Reply:
Sure, I'll add a comment for the magic number